scripts/watch.ts

```diff
@@ -45,14 +45,14 @@ const main = async () => {
   );
 
   // Do an initial build, which will also populate the Turborepo cache
-  await $`yarn turbo build --output-logs=new-only --filter="@pcd/*"`;
+  await $`time yarn tsc -b tsconfig.cjs.json tsconfig.esm.json`;
```

Comment:
Turbo prints out how long it takes to build and `tsc` doesn't. I don't mind not doing it, but it was useful to be able to see how long builds took in various circumstances (small changes, large changes, "warm" builds and builds of fresh checkouts or just after `yarn clean`).

Reply:
I think I'd like to remove this as I believe it would be spam.

<img width="675" alt="Screenshot 2024-01-15 at 1 50 07 PM" src="https://github.com/proofcarryingdata/zupass/assets/2636237/dde1df3d-8235-4086-883b-7a558fe38b31">